simde/x86/sse4.2.h

```diff
@@ -37,6 +37,204 @@ SIMDE__BEGIN_DECLS
 #  define SIMDE_X86_SSE4_2_ENABLE_NATIVE_ALIASES
 #endif
 
+SIMDE__FUNCTION_ATTRIBUTES
+int
+simde_mm_cmpestra(simde__m128i a, int la, simde__m128i b, int lb, const int imm8){
+#if defined(SIMDE_X86_SSE4_2_NATIVE)
+  return _mm_cmpestra(a, la, b, lb, imm8);
+#else
+  simde__m128i_private
+  BoolRes_ = simde__m128i_to_private(simde_mm_setzero_si128()),
+  a_ = simde__m128i_to_private(a),
+  b_ = simde__m128i_to_private(b);
+  if(imm8&1){
+    int UpperBound = (128 / 16) - 1;
```

Comment:
> The compiler will emit an error instead of the condition always being true. If you want a great example of this: https://freedom-to-tinker.com/2013/10/09/the-linux-backdoor-attempt-of-2003/

I have made upper_bound as const. This was very interesting read :D